tutorials/plugins/running_code_in_the_editor.rst

```diff
@@ -245,6 +245,147 @@ angle add a setter ``set(new_speed)`` which is executed with the input from the
     to run in the editor too. Autoload nodes cannot be accessed in the editor at
     all.
 
+Getting notified about Resources changing
+-----------------------------------------
```

Comment:
```suggestion
Getting notified when resources change
--------------------------------------

```

Reply:
Hm I personally like to capitalise Keywords, and in this case it is referring to a class. But I have just looked around the wiki and that seems to not be common practce.